internal/database/export_test.go

```diff
@@ -390,17 +390,25 @@ func TestKeysInBatch(t *testing.T) {
 
 	// Re-fetch the ExposureBatch by leasing it; this is important to this test which is trying
 	// to ensure our dates are going in-and-out of the database correctly.
-	eb, err := testDB.LeaseBatch(ctx, time.Hour, now)
+	leased, err := testDB.LeaseBatch(ctx, time.Hour, now)
 	if err != nil {
 		t.Fatal(err)
 	}
 
-	// Lookup the keys; they must be only the key created_at the endTimestamp
-	// (because start is exclusive, end is inclusive).
+	// Check that the batch times from the database are *exactly* what we started with.
```

Comment:
Jason, if I understood our conversation last night correctly, the root cause of your problem was not Postgres/Go time mismatch. So I don't think you need these checks, which I suggested while in paranoid mode.

Reply:
Yes agree, but there's no harm in leaving them - e.g., might help someone replatforming this code in the future.